espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/ElementValue.kt

```diff
@@ -35,16 +38,32 @@ class ElementValue(private val isReplacing: Boolean) : RequestHandler<ElementVal
                 return null
             }
         } catch (e: NumberFormatException) {
-            throw InvalidArgumentException(String.format("Cannot convert '%s' to an integer",
-                    params.value))
+            throw InvalidArgumentException(String.format("Cannot convert '$value' to an integer"))
         }
 
         val viewInteraction = Element.getViewInteractionById(elementId)
         if (isReplacing) {
             viewInteraction.perform(replaceText(value))
         } else {
-            viewInteraction.perform(typeText(value))
+            try {
```

Comment:
is this block of code duplicated?

Reply:
A part of.
Will refactor as another PR (SendKeys.kt)